01LessonOne/index.html

```diff
@@ -1,2 +1,3 @@
 Your homework for today is take a pre-work review.
```

Comment:
sorry, i think i changed the file and didn't commit before we started the class project.